sdk/golang/ccnp/quote/quote.go

```diff
@@ -0,0 +1,191 @@
+/*
+* Copyright (c) 2023, Intel Corporation. All rights reserved.<BR>
+* SPDX-License-Identifier: Apache-2.0
+ */
+
+package quote
+
+import (
+	"bytes"
+	"context"
+	"encoding/base64"
+	"encoding/binary"
+	"log"
+	"strings"
+	"time"
+
+	pb "github.com/intel/confidential-cloud-native-primitives/sdk/golang/ccnp/quote/proto"
+	pkgerrors "github.com/pkg/errors"
```

Comment:
why not use golang standard errors pkg?

Reply:
> why not use golang standard errors pkg?
pkg/errors supports adding context to errors.